### ISSUE TITLE
testthat fails on test-DAISIE_SR_code and other test failures

### ISSUE THREAD

**Issue by Neves-P** [rsetienne/DAISIE#77]
`testthat::expect_equal(unname( sims[[1]][[1]]$stt_all[26, ] ), c(0, 56, 11, 0, 66))` at test-DAISIE_SR_code at line 20 fails locally. Somehow this passes on travis, see https://travis-ci.org/rsetienne/DAISIE/builds/585569853

Unsure how this is happening. Further, I get 5 warnings locally such as this on `test-integration_DAISIE`
```
test-integration_DAISIE.R:150: warning: The parameter choice for 2type DAISIE_ML works
longer object length is not a multiple of shorter object length
```

Will try to sort this out tomorrow

**Comment by richelbilderbeek** [comment]
I predict this has to do with RNG: on Travis, the tests are run in consecutive order, resulting in the same (unstable!) chain of RNG states.

Tip: find the two seeds, one that does and one that does not give the warnings. In one of two cases, the input is checked too little. Make the sim run either silently for both seeds, or expect an error in one case.

Good luck :+1:

**Comment by Neves-P** [comment]
This was fixed by updating R to 3.6 due to RNG changes, see [here](https://stat.ethz.ch/pipermail/r-announce/2019/000641.html). Thanks a lot to @thauffe @richelbilderbeek and @rsetienne for help!
